website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+---
+id: user_attributes
+title: User Attributes
+---
+
+## User attributes
+
+This section defines the attributes that can be made available to services
+connected to Check-in.
+
+### 1. Unique ID
+
+<!-- markdownlint-disable line-length no-inline-html -->
+
+|          attribute name | Unique ID                                                                                       |
+| ----------------------: | :---------------------------------------------------------------------------------------------- |
+|         **description** | An unique identifier for the user                       |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.5923.1.1.1.13` (eduPersonUniqueId)                                         |
+|          **OIDC scope** | `openid`                                                                                        |
+|       **OIDC claim(s)** | `sub`                                                                                           |
+| **OIDC claim location** | <ul><li>ID token</li><li>Userinfo endpoint</li><li>Introspection endpoint</li></ul>             |
+|              **origin** | Check-in assigns this attribute on user registration                                            |
+|             **changes** | No                                                                                              |
+|        **multiplicity** | No                                                                                              |
+|        **availability** | Always                                                                                          |
+|             **example** | _4a58cb0338fb68060ba75e20e02a4844148f9c63a5cd9a7418dc5a9a6ab641cbaef73b4c65519286359f766c45bdbad36cc9189e75a03bd12f13526c34cab93e@eid-proxy.aai.grnet.gr_   |
+|               **notes** | Use **EGI ID** within your application as the unique-identifier key for the user                |
+|              **status** | Stable                                                                                          |
+
+<!-- markdownlint-enable line-length no-inline-html -->
+
+### 2. Person Identifier
+
+|          attribute name | Person Identifier                                                                   |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | The user's person identifier                                                        |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/PersonIdentifier` (PersonIdentifier) |
```

Comment:
Question: Is this forwarded by the eID Proxy to SAML2int SPs as is or mapped to an eduPerson attribute?

Reply:
It is forwarded as is

Reply:
We currently have no SAML SPs connected that get legal person attributes so it is null for now